app/views/shared/_intercom.erb

```diff
@@ -0,0 +1,4 @@
+<script type="text/javascript">
```

Comment:
I'm not sure it matters much in a partial, but this should probably be named `_intercom.html.erb`.